saleor/order/models.py

```diff
@@ -366,8 +366,12 @@ class OrderNote(models.Model):
     order = models.ForeignKey(
         Order, related_name='notes', on_delete=models.CASCADE)
     content = models.CharField(max_length=250)
+    is_public = models.BooleanField(default=True)
+
+    class Meta:
+        ordering = ('-date',)
 
     def __str__(self):
-        return pgettext_lazy(
+        return str((pgettext_lazy(
```

Comment:
I think you can just use `pggettext` instead of `pgettext_lazy`

Reply:
@Pacu2 is right. The goal of `*_lazy` is not to return a string immediately so that the result can depend on the locale activated at a later time (mostly useful for expressions evaluated at import time).